bcgov/von-bc-registries-agent
# Fixes for withdrawn and amalgamations

Fixes for processing of withdrawn and amalgamated corp types.

Updates to audit processing and audit fixes.

Note this PR will require delete and reload of the audit table:

- in event processor database:  "delete from corp_audit_log"
- run script "bcreg/populate_audit_table.py"

To test these changes (local):

1. shutdown cron jobs
2. reload audit table (as above)
3. run "bcreg/display_processed_corps_counts.py" to get numbers
4. run "bcreg/display_processed_corps_counts.py --fixme"
5. run "bcreg/bc_reg_pipeline_initial_load.py" to load and stage credentials
6. run "bcreg/bc_reg_pipeline_post_credentials.py" to post credentials
7. run "bcreg/display_processed_corps_counts.py" to get numbers - should all be ok
8. run "bcreg/bc_reg_pipeline.py" followed by "bcreg/display_processed_corps_counts.py" a few times
9. if there are any missing or incorrect corps, run steps 4-7 and it should fix them

Once everything is ok re-start the cron jobs.
